### ISSUE TITLE
Errors and warnings printed out in terminal

### ISSUE THREAD

**Issue by pietrobalatti** [ADVRHumanoids/XCM#27]
@liesrock until commit a67f2296e1ed714388233dc1c15131073de88120 everything works fine, from the following one (e68e0a2f52cc1b1045fd346b6fccfea8a63520fd) I started having warnings and errors repeatedly printed out in the terminal. These is what I see in the NRTDeployer terminal:

```
ERROR while parsing CommandAdvr message: joint LHipLat undefined
WARNING in getIndex! Joint LHipYaw is not defined inside the ROS controller!
ERROR while parsing CommandAdvr message: joint LHipYaw undefined
WARNING in getIndex! Joint LHipSag is not defined inside the ROS controller!
ERROR while parsing CommandAdvr message: joint LHipSag undefined
WARNING in getIndex! Joint LKneeSag is not defined inside the ROS controller!
ERROR while parsing CommandAdvr message: joint LKneeSag undefined
WARNING in getIndex! Joint LAnkSag is not defined inside the ROS controller!
ERROR while parsing CommandAdvr message: joint LAnkSag undefined
WARNING in getIndex! Joint LAnkLat is not defined inside the ROS controller!
ERROR while parsing CommandAdvr message: joint LAnkLat undefined
WARNING in getIndex! Joint RHipLat is not defined inside the ROS controller!
ERROR while parsing CommandAdvr message: joint RHipLat undefined
WARNING in getIndex! Joint RHipYaw is not defined inside the ROS controller!
ERROR while parsing CommandAdvr message: joint RHipYaw undefined
WARNING in getIndex! Joint RHipSag is not defined inside the ROS controller!
ERROR while parsing CommandAdvr message: joint RHipSag undefined
WARNING in getIndex! Joint RKneeSag is not defined inside the ROS controller!
ERROR while parsing CommandAdvr message: joint RKneeSag undefined
WARNING in getIndex! Joint RAnkSag is not defined inside the ROS controller!
ERROR while parsing CommandAdvr message: joint RAnkSag undefined
WARNING in getIndex! Joint RAnkLat is not defined inside the ROS controller!
ERROR while parsing CommandAdvr message: joint RAnkLat undefined
WARNING in getIndex! Joint WaistLat is not defined inside the ROS controller!
ERROR while parsing CommandAdvr message: joint WaistLat undefined
WARNING in getIndex! Joint WaistSag is not defined inside the ROS controller!
ERROR while parsing CommandAdvr message: joint WaistSag undefined
WARNING in getIndex! Joint WaistYaw is not defined inside the ROS controller!
ERROR while parsing CommandAdvr message: joint WaistYaw undefined
WARNING in getIndex! Joint LShSag is not defined inside the ROS controller!
ERROR while parsing CommandAdvr message: joint LShSag undefined
WARNING in getIndex! Joint LShLat is not defined inside the ROS controller!
ERROR while parsing CommandAdvr message: joint LShLat undefined
WARNING in getIndex! Joint LShYaw is not defined inside the ROS controller!
ERROR while parsing CommandAdvr message: joint LShYaw undefined
WARNING in getIndex! Joint LElbj is not defined inside the ROS controller!
ERROR while parsing CommandAdvr message: joint LElbj undefined
WARNING in getIndex! Joint LForearmPlate is not defined inside the ROS controller!
ERROR while parsing CommandAdvr message: joint LForearmPlate undefined
WARNING in getIndex! Joint LWrj1 is not defined inside the ROS controller!
ERROR while parsing CommandAdvr message: joint LWrj1 undefined
WARNING in getIndex! Joint LWrj2 is not defined inside the ROS controller!
ERROR while parsing CommandAdvr message: joint LWrj2 undefined
WARNING in getIndex! Joint NeckYawj is not defined inside the ROS controller!
ERROR while parsing CommandAdvr message: joint NeckYawj undefined
WARNING in getIndex! Joint NeckPitchj is not defined inside the ROS controller!
ERROR while parsing CommandAdvr message: joint NeckPitchj undefined
WARNING in getIndex! Joint RShSag is not defined inside the ROS controller!
ERROR while parsing CommandAdvr message: joint RShSag undefined
WARNING in getIndex! Joint RShLat is not defined inside the ROS controller!
ERROR while parsing CommandAdvr message: joint RShLat undefined
WARNING in getIndex! Joint RShYaw is not defined inside the ROS controller!
ERROR while parsing CommandAdvr message: joint RShYaw undefined
WARNING in getIndex! Joint RElbj is not defined inside the ROS controller!
ERROR while parsing CommandAdvr message: joint RElbj undefined
WARNING in getIndex! Joint RForearmPlate is not defined inside the ROS controller!
ERROR while parsing CommandAdvr message: joint RForearmPlate undefined
WARNING in getIndex! Joint RWrj1 is not defined inside the ROS controller!
ERROR while parsing CommandAdvr message: joint RWrj1 undefined
WARNING in getIndex! Joint RWrj2 is not defined inside the ROS controller!
ERROR while parsing CommandAdvr message: joint RWrj2 undefined
WARNING in getIndex! Joint LHipLat is not defined inside the ROS controller!


WARNING in read_sensors: no sensor corresponding to link  is present in given URDF/SRDF! Check that FTmsg.header.frame_id is the parent link of the sensor name in URDF! Also check that FT fixed joints are defined inside the force_torque_sensors SRDF group
WARNING in read_sensors: no sensor corresponding to link  is present in given URDF/SRDF! Check that IMUmsg.header.frame_id is the parent link of the sensor name in URDF! Also check that IMU fixed joints are defined inside the force_torque_sensors SRDF group

```

**Comment by liesrock** [comment]
Hi @pietrobalatti,
usually it is caused by the CommunicationHandler not properly started or a different config file loaded for the CommunicationHandler and the NRTDeployer.

Can you double check?

**Comment by pietrobalatti** [comment]
I double checked and the config file is always the same. In my specific case advr-superbuild/configs/ADVR_shared/bigman/configs/config_walkman_floating_base.yaml

**Comment by liesrock** [comment]
Ok, can you report also the output of the CH?

**Comment by pietrobalatti** [comment]
Same error output in CH an XBotGUI terminals

**Comment by liesrock** [comment]
FIxed in 	bb93ca8
